runtime/oti/j9nonbuilder.h

```diff
@@ -3585,6 +3585,28 @@ typedef struct J9Method {
 	U_8* bytecodes;
 	struct J9ConstantPool* constantPool;
 	void* methodRunAddress;
+
+	/**
+	 * This field is overloaded and can have several meanings:
+	 * 
+	 * 1. If the `J9_STARTPC_NOT_TRANSLATED` bit is set, this method is interpreted
+	 * 		a. If the value is non-negative, then the field contains the invocation count which is decremented towards
+	 * 		zero. When the invocation count reaches zero, the method will be queued for JIT compilation. The invocation
+	 * 		count is represented as:
+	 * 
+	 * 		```
+	 * 		(invocationCount << 1) | J9_STARTPC_NOT_TRANSLATED
+	 * 		```
+	 * 
+	 * 		b. If the value is negative, then it can be one of:
+	 * 			- `J9_JIT_NEVER_TRANSLATE`: ???
```

Comment:
This seems to be never used or set. @gacholio do you might have an idea of what this was used for? I don't see us setting this value so should we remove it?

Reply:
This does get set, eg
https://github.com/eclipse/openj9/blob/250b180e1fa101e3dd3e171a3bb686b9301eab08/runtime/codert_vm/CodertVMHelpers.cpp#L185

Reply:
Good catch. Not sure how I missed it. Seems we call `jitMethodFailedTranslation` when we failed the first compilation of a method. Where is this value consumed though?

Reply:
The interpreter folks would be able to say for sure, but I _think_ if the value is negative the VM won't decrement the count, and therefore because the low bit is set, the VM will never trigger a compilation request.

Reply:
@tajila could you shed some light on what happens if the `extra` field is `J9_JIT_NEVER_TRANSLATE `?

Reply:
@dsouzai is correct.

Reply:
Fixed in [force-push](https://github.com/eclipse/openj9/compare/9837dbd2dc060a8d9102441fffcc446c13902523..8ea8c5df617546e5fb6907f41e50112adad4e8e6).